### PR TITLE
release(v0.1.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.1.0-alpha.0 release.

Signed-off-by: Artem Chernyshev <artem.0xD2@gmail.com>

### CODE REVIEW

**Comment by smira** [comment]
as we're on it, it probably makes sense to fix one more thing in Kres:

`.conform.yaml`:

```
      scopes: ["*"]
```


this actually breaks conform, as it expects it to be a regexp (as you can see, conform on your PR doesn't report a status)

so easy fix to change it to `.*`

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+In that case it fetches servers and cluster from the management clusters, is able to
+get kubeconfig and talosconfig for the deployed clusters and can also show nodes and pods in a cluster,
+as well as services in the node.
+
+
+### Contributors
+
+* Spencer Smith
+* Andrew Rynhard
+* Andrey Smirnov
```
Comment (andrewrynhard) [inline CHANGELOG.md:25]: I did something? 😆

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Unix4ever) [reply CHANGELOG.md:25]: well, at least you created this repo :)

File: hack/release.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+previous = "v0.0.1"
+pre_release = true
+
+[notes]
+
+  [notes.intro]
+
+    title = "The First Release"
+    description = """\
+This is the first release of the UI for Talos and Sidero.
```
Comment (Unix4ever) [inline hack/release.toml:17]: Please review this description here.

**Review by andrewrynhard — APPROVED** [review]

File: hack/release.toml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+[notes]
+
+  [notes.intro]
+
+    title = "The First Release"
+    description = """\
+This is the very first release of the UI for Talos and Sidero.
+
+First iteration is only able to run against management cluster with Sidero installed.
```
Comment (andrewrynhard) [inline hack/release.toml:19]:
```suggestion
The first iteration is only able to run against the management cluster with Sidero installed.
```

File: hack/release.toml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Theila gets credentials from `~/.kube/config` or from the file defined in `KUBECONFIG`
+environment variable. And it is always using the current context.
```
Comment (andrewrynhard) [inline hack/release.toml:21]:
```suggestion
environment variable. 
The current context is always used.
```

File: hack/release.toml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+For each Sidero Kubernetes cluster it is possible to:
+- get kubeconfig and talosconfig.
```
Comment (andrewrynhard) [inline hack/release.toml:24]:
```suggestion
- get the kubeconfig and talosconfig.
```

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Unix4ever) [reply CHANGELOG.md]: I've updated changelog. Looks like if you don't set the previous tag it also works and the output doesn't include all dependencies updates.

**Review by smira — APPROVED** [review]

**Pull request merged**